ui-v2/tests/integration/services/dc-test.js

```diff
@@ -0,0 +1,45 @@
+import { moduleFor, test } from 'ember-qunit';
+import { skip } from 'qunit';
+import repo from 'consul-ui/tests/helpers/repo';
+const NAME = 'dc';
+moduleFor(`service:${NAME}`, `Integration | Service | ${NAME}s`, {
+  // Specify the other units that are required for this test.
+  needs: [
+    'service:settings',
+    'service:store',
+    `adapter:${NAME}`,
+    `serializer:${NAME}`,
+    `model:${NAME}`,
+    // relationships
+    'model:service',
+    'model:node',
+  ],
+});
+skip("findBySlug (doesn't interact with the API) but still needs an int test");
```

Comment:
I'm skipping this one for now as this method doesn't interact with the API, but I'll get reminded to come back here at some point.